Jenkinsfile

```diff
@@ -0,0 +1,151 @@
+@Library(value='pipeline-library@master', changelog=false)
+import com.genesys.jenkins.Service
+ 
+final serviceBuild = new Service()
+
+pipeline {
+
+    agent any
+
+    tools {
+        maven 'Maven 3.5.3'
+        jdk 'OpenJDK 1.8.0 (latest)'
+    }
+
+    environment {
+        NODE_VERSION = '11.15.0'
+        SNYK_TOKEN = credentials("PureCloudSnykToken")
+    }
+
+    stages {
+        stage("pre-build") {
+            steps {
+                copyArtifacts(projectName: 'build-github-swagger-codegen-sdkv1', filter: 'modules/swagger-codegen-cli/target/swagger-codegen-cli.jar', target: 'swagger-codegen', fingerprintArtifacts: true);
+
+                checkout scm
+
+                dir('platform-client-sdk-internal-config') {
+                    git url: "${scm.getUserRemoteConfigs()[1].getUrl()}"
```

Comment:
Use the repo URL explicitly instead of a reference to the job config.

Reply:
I had suggested hiding references to our internal git repos per your suggestion on my CLI SDK PR